templates-java/src/test/resources/templates/config/template-producer-message-test.txt

```diff
@@ -0,0 +1,39 @@
+package br.com.example;
+
+import com.amazonaws.services.sqs.AmazonSQSAsync;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.cloud.aws.messaging.core.QueueMessagingTemplate;
+import org.springframework.cloud.aws.messaging.core.SqsMessageHeaders;
+import org.springframework.stereotype.Component;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+@Component
+public class ProducerMessage {
+
+    @Value("${cloud.aws.sqs.queue-name}")
```

Comment:
The `@Value` annotation is being used to inject a property value. Consider using a configuration class or a properties file to manage the configuration of the queue name.